third_party/versions.bzl

```diff
@@ -132,7 +131,7 @@ HTTP_DEPENDENCY_VERSIONS = {
         "version": "v1.5.0-alpha-2",
     },
     "rules_proto": {
-        "sha": "e0cab008a9cdc2400a1d6572167bf9c5afc72e19ee2b862d18581051efab42c9",
+        "sha": "602e7161d9195e50246177e7c55b2f39950a9cf7366f74ed5f22fd45750cd208",
```

Comment:
Why this change? It seems unrelated--I suggest reverting this part.

Reply:
Reverted.